extension/docs/CancelRefundPayment.md

```diff
@@ -41,10 +41,12 @@
           ]
         }
         ```
-    Both transaction types work the same, because Adyen will always pick the right action for the current transaction.
-    If the transaction is not authorized yet, Adyen will try to cancel.
-    If the transaction is already charged, Adyen will do refund.
-1. In `Payment.transactions`, extension module finds the first transaction with `type='Charge' and state='Success'`. 
+
+- Both transaction types work the same, because Adyen will always pick the right action for the current transaction.
```

Comment:
```suggestion
- Both charge and refund is processed by Adyen in a similar manner and it will automatically pick the right action for the current transaction.
```